src/DGtal/dec/doc/modulePolygonalCalculus.dox

```diff
@@ -200,7 +200,29 @@ Operator  | Output | Description
 `calculus.M(f,lambda)`  | \f$ n_f\times n_f\f$ | Inner product operator on discrete 1-forms operator (lambda is a regularization parameter, see @cite degoes2020discrete)
 `calculus.LaplaceBeltrami(f,lambda)`|  \f$ n_f \times n_f\f$  | Laplace-Beltrami operator \f$ D^tMD \f$ (lambda is a regularization parameter, see @cite degoes2020discrete), the matrix is PSD
 
- 
+\subsection sectcovOPL Covariant Operators and Vector-Laplacian
+
+\subsubsection sectcovOP Covariant Gradient and Projection
+
+Since these operators are themselves matrices they cannot be constructed in the same way as the others ( Operator builder Matrix * function restricted to face Vector ), you need the specify the local vector field as well, in the same format as in @cite degoes2020discrete,
+i.e. 
+\f{eqnarray*}{
+    u_{f} = [u_{v_1}^t \ldots u_{v_{nf}}^t]^t
+\f}
+where \f$u_{v_i}\f$ are the intrinsic vectors at each adjacent vertex expressed in their own tangent basis.
+
+Operator  | Output | Description
+------------- | ------------- | ----
+`calculus.CovariantGradient(f,uf)`  | \f$ 2\times 2\f$ | Covariant Gradient defined for intrinsic vector valued 0-forms at face f
+`calculus.CovariantProjection(f,uf)`|  \f$ n_f \times 2\f$  | Covariant Projection operator defined for intrinsic vector valued 0-forms at face f
+
+\subsubsection sectVL Vector Laplacian
+
+Using the same format for vector fields as above, we can define a Vector Laplacian operator per face.
+Operator  | Output | Description
+------------- | ------------- | ----
+`calculus.CovL(f,lambda)`  | \f$ 2n_f\times 2n_f\f$ | Vector Laplacian defined as the associated Matrix with the Dirichlet energy for vector valued 0-forms (lambda is a regularization parameter, see @cite degoes2020discrete) at face f, PSD matrix 
+
 
```

Comment:
+ add the transport operators ? with an illustration on a "single face" as above?